reframe/frontend/cli.py

```diff
@@ -229,6 +238,17 @@ def main():
     # Init modules system
     init_modules_system(system.modules_system)
 
+    if options.module_map_file:
+        get_modules_system().load_mapping_from_file(options.module_map_file)
+
+    if options.module_map_cl:
+        for m in options.module_map_cl:
+            try:
+                get_modules_system().load_mapping(m)
+            except Exception as e:
+                raise ConfigError('Invalid format of the command-line'
+                                  'module mapping: %s\n %s' % (m, e))
```

Comment:
In case of error you should exit.

Reply:
- Generally, both of the above calls (loading from file and from the command line arguments) you could put them in a `try/except` block.
- The error message should read as (_lowercase_): `"could not set module mapping: %s" % e`